### PR TITLE
Added unbuffer to the bridge and sim launch commands

### PR DESCRIPTION
**Pull request by nkoenig**
Resolves issue #945.

Signed-off-by: Nate Koenig <nate@openrobotics.org>

### CODE REVIEW

**Comment by peci1** [comment]
The unbuffer call has to go in front of the `ign launch` command, not in front of the `tee`.

File: docker/cloudsim_bridge/run_bridge.bash [hunk 1]
```diff
@@ -21,4 +21,4 @@ export ROSCONSOLE_STDOUT_LINE_BUFFERED=1
 # atop process monitoring
 atop -R -w /home/developer/.ros/atop_log &
 
-ign launch cloudsim_bridge.ign -v 4 "$@" 2>&1 | tee /home/developer/.ros/bridge.log
+unbuffer -p ign launch cloudsim_bridge.ign -v 4 "$@" 2>&1 | tee /home/developer/.ros/bridge.log
```
Comment (iche033) [inline docker/cloudsim_bridge/run_bridge.bash:24]: I think you don't need the `-p` if the `unbuffer` command is at the front, i.e. not after the pipe

**Comment by nkoenig** [comment]
> The unbuffer call has to go in front of the `ign launch` command, not in front of the `tee`.

I think it's not fixed.

File: docker/cloudsim_bridge/run_bridge.bash (hunk 1 above)
Comment (nkoenig) [reply docker/cloudsim_bridge/run_bridge.bash]: Thanks. It has been removed.

**Review by iche033 — APPROVED** [review]

**Pull request merged**